integration_test/robots/profile_information_robot.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:fluffychat/pages/chat_details/chat_details_view.dart';
+import 'package:fluffychat/pages/profile_info/profile_info_body/profile_info_contact_rows.dart';
+import 'package:fluffychat/pages/profile_info/profile_info_body/profile_info_header.dart';
+import 'package:fluffychat/widgets/avatar/avatar.dart';
+import 'package:fluffychat/widgets/twake_components/twake_icon_button.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:patrol/patrol.dart';
+import '../base/core_robot.dart';
+
+class ProfileInformationRobot extends CoreRobot {
+  ProfileInformationRobot(super.$);
+
+  PatrolFinder getBackIcon() {
+    return $(AppBar).$(TwakeIconButton);
+  }
+
+  PatrolFinder getTitle() {
+    return $(AppBar).$(Text);
+  }
+
+  PatrolFinder getAvatar() {
+    return $(ProfileInfoHeader).$(Avatar);
+  }
+
+  PatrolFinder getDisplayName() {
+    final count = $(ProfileInfoHeader).$(Text).evaluate().length;
+    if(count>2){
+      return $(ProfileInfoHeader).$(Text).at(count-2);
```

Comment:
```suggestion
    if (count>2) {
      return $(ProfileInfoHeader).$(Text).at(count-2);
```

We do not have a checkstyle equivalent for this project?

Cc @hoangdat